src/blas/backends/portblas/portblas_batch.cxx

```diff
@@ -695,7 +710,7 @@ sycl::event gemm_batch(sycl::queue &queue, oneapi::mkl::transpose *transa,
                        const double **b, std::int64_t *ldb, double *beta, double **c,
                        std::int64_t *ldc, std::int64_t group_count, std::int64_t *group_size,
                        const std::vector<sycl::event> &dependencies) {
-    throw unimplemented("blas", "gemm_batch", " for USM");
+    throw unimplemented("blas", "gemm_batch", " for USM using double");
```

Comment:
Could you explain me why it doesn't work with `double`? I thought that in portBLAS if it works with `float` it works with `double`. Did you test it?

Reply:
It can probably be added, I'm looking this. I have found that the gemm_batch tests only use `group_count=5`. I need to test this with `group_count=1`.